src/administration/install_docker.md

```diff
@@ -1,78 +1,118 @@
 # Docker Installation
 
-Make sure you have both docker and docker-compose(>=`1.24.0`) installed. On Ubuntu, just run `apt install docker-compose docker.io`. Next,
+Lemmy may be run within a Docker container. This is the easiest way to get started with Lemmy, however it still requires some technical knowledge.
+
+## Prerequisites
+
+A working knowledge of Docker is required to deploy Lemmy using this method. For the unfamiliar with Docker, please refer to the [official documentation](https://docs.docker.com/).
+
+Ensure that a modern version of `docker` and `docker-compose` (>=`1.24.0`) are available in the environment. On Ubuntu, this is accomplished by executing `apt install docker-compose docker.io`; other distributions may have different package names.
+
+### Reverse Proxy / Webserver
+
+Lemmy requires the use of a Reverse Proxy to properly direct requests to either the Lemmy server or Lemmy UI. This is because Lemmy is composed of two separate components, the server and the UI, which are served from different containers (or services in Docker Swarm mode) and different ports. The reverse proxy is responsible for directing requests to the correct component based on the URL, and also provides SSL termination for the proxied service.
+
+Some common choices include webservers such as [NGINX](https://www.nginx.com/), or a dedicated proxy such as [Traefik](https://traefik.io/). Configuration is _not_ limited to these two options, however these are the most common and example configurations are provided for them.
+
+The Reverse proxy may be part of the same Docker Compose file as the Lemmy server and UI, or it may be a separate service. In a production configuration it is recommended that the reverse proxy be a separate service, as this allows for greater flexibility in scaling and configuration.
+
+## Deployment
+
+Create a location to store the Lemmy docker environment; set any required permissions.
 
 ```bash
 # Create a folder for the lemmy files. the location doesnt matter, you can put this anywhere you want
-mkdir lemmy
-cd lemmy
+mkdir -p /data/lemmy/{volumes,config}
+mkdir -p /data/lemmy/volumes/{pictrs,postgres}
+chown 991:991 /data/lemmy/volumes/pictrs
+cd /data/lemmy
```

Comment:
I would consider using the same dir as ansible for consistency which is `/srv/lemmy/{domain}`. And still mention that any path can be used.

Reply:
I've implemented this, but would prefer that somewhere other than `/srv` be used - while `/srv` is part of the FHS not all Linuxes are FHS-compliant and Gentoo, for example, doesn't have a `/srv`. Any suggestions for alternatives?

As a bit of an aside this may be a good opportunity to simplify both guides - how many users are actually hosting multiple lemmy instances on a single system? I have enough trouble with the resource utilisation for a very small instance on a VPS...

My point is that any users  in a position to run multiple instances on a single server should have the prerequisite experience to know when they need to deviate from the provided instructions while the vast majority of users following this guide will not need anything like a multi-domain level of complexity.